### PR TITLE
Add of -c- group, EcoRegion Level 3 Zone 21 group

### LINKED ISSUE #8: Move the LS_C2_SRST:nw-poudre-historical branch to this repo
And add it in as a target between a & b.

### PR DESCRIPTION
**Pull request by steeleb**
Hey Katie!

This one is just like the -b- group, but grabbing polygon centers across all >1ha lakes/reservoirs in the EcoRegion 3 Zone 21 AOI.  This looks like a lot of LOC, but most of it is copy/paste of the `c_regional_RS_data_acquisition/py/runGEEperTile.py` (1700 LOC) other than some filepath changes.

On that note, all files in `c_regional_RS_data_acquisition/py/` directory are the same as those in the -b- group of the same subfolder name BUT! the directory paths have been changed. This is one of the things that I couldn't figure out how to automate/generalize when running multiple configs in one repository. Alas. 

Please make sure tar_make(a_aoi_centers_to_csv) runs for you (this is what creates the location file to be run in EE), otherwise the EE run is the same as the -b- group. Obviously if you want to poke through the -c- group, go for it, but same as last time, please run the EE pull using your credentials if you do!

This closes #8

### CODE REVIEW

File: c_regional_RS_data_acquisition.R [hunk 1]
```diff
@@ -0,0 +1,157 @@
...
+    command = yaml_file,
+    read = read_yaml(!!.x),
+    packages = "yaml",
+    cue = tar_cue("always")
+  ),
+  
+  # load, format, save yml as a csv
+  tar_target(
+    name = c_yml,
+    command = format_yaml(yaml = c_config_file,
```
Comment (kathryn-willi) [inline c_regional_RS_data_acquisition.R]: format_yaml may benefit having the following code ahead of the write_csv()?

` if(!dir.exists(file.path(parent_path, "run/"))) {dir.create(file.path(parent_path, "run/"), recursive = TRUE)}`

**Comment by kathryn-willi** [comment]
I don't see anything worth commenting on, really! 

As far as how to deal with multiple configs that are causing the duplicate workflow... doing a quick Google search brought me to os and pathlib? Not sure if they'd actually be worth looking into (if you haven't already!) or if they'd even be applicable tho!

Otherwise everything ran smoothly. I've officially run all steps (a thru c) and stored their outputs in my own Google account successfully 😄

**Review by kathryn-willi — APPROVED** [review]

File: c_regional_RS_data_acquisition.R (hunk 1 above)
Comment (steeleb) [reply c_regional_RS_data_acquisition.R]: Ah! You caught a small annoyance of targets. I needed to add the check_architecture target to the command here. I think it's funny that I can't just tell targets to run something 'first and always' - even adding tar_cue("always") doesn't give this priority which BAFFLES ME. Fixing now.

Comment (steeleb) [reply c_regional_RS_data_acquisition.R]: Okay, @kathryn-willi, I added 'x_check_dir_structure' to the `command` argument of the targets for yaml and locs formatting, and now this should be g2g! Thanks again for noticing this!

**Pull request merged**